pubspec.yaml

```diff
@@ -1,5 +1,11 @@
 name: fluri
 version: 0.0.0
 description: 'Fluri is a fluent URI library built to make URI mutation easy'
+authors:
```

Comment:
We can add the client platform team email later once an @workiva.com one has been setup